swagger.yaml

```diff
@@ -190,6 +190,36 @@ paths:
         "400":
           description: bad input parameter
 
+  /internal-transactions:
+    get:
+      summary: get internal transactions
+      operationId: get_internal_transactions
+      responses:
+        "200":
+          description: internal transactions
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - items
+                  - next_page_params
+                properties:
+                  items:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/InternalTransaction"
+                  next_page_params:
+                    type: object
+                    example:
+                      {
+                        "block_number": 27170298,
+                        "index": 0,
+                        "transaction_index": 0
+                      }
+        "400":
+          description: bad input parameter
```

Comment:
I think, it's impossible to get 400 in that endpoint, wdyt?

Reply:
You're right. I removed 400 status https://github.com/blockscout/blockscout-api-v2-swagger/pull/25/commits/76f69d7e52b71a1c7db6e52c9c4314495cd4a45f.